lib/ansible/modules/cloud/digital_ocean/digital_ocean_image_facts.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: digital_ocean_image_facts
+short_description: Gather facts about DigitalOcean images
+description:
+    - This module can be used to gather facts about DigitalOcean provided images.
+    - These images can be either of type C(distribution), C(application) and C(private).
+author: "Abhijeet Kasurde (@akasurde)"
+version_added: "2.5"
+options:
+  image_type:
+    description:
+     - Specifies the type of image facts to be retrived.
+     - If set to C(application), then facts are gathered related to all application images.
+     - If set to C(distribution), then facts are gathered related to all distribution images.
+     - If set to C(private), then facts are gathered related to all private images.
+     - If not set to any of above, then facts are gathered related to all images.
+    default: 'all'
+    choices: [ 'all', 'application', 'distribution', 'private' ]
+    required: false
+  oauth_token:
```

Comment:
A few days ago **utils/module_docs_fragments/digital_ocean.py** (Ansible Digital Ocean utils) was updated in PR #35934 which introduced generic configurations and documentation that should be used, particularly here this documentation fragment **extends_documentation_fragment: digital_ocean.documentation** (https://github.com/Akasurde/ansible/blob/devel/lib/ansible/utils/module_docs_fragments/digital_ocean.py)